.eslintrc.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    node: true,
+  },
+  extends: 'airbnb',
+  globals: {
+    Atomics: 'readonly',
+    SharedArrayBuffer: 'readonly',
+  },
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 2018,
+    sourceType: 'module',
+  },
+  plugins: [
+    'react',
+  ],
+  rules: {
+  },
+};
```

Comment:
@stellaliang21 @mishafrenkel I've seen this exact config file from at least 2 other students at this point. I'm not sure where it's from, but I believe most of it is unnecessary. Careful about allowing unnecessary code into a project. It can have unintended side-effects that are difficult to debug.

Reply:
@jordanstholmes Not sure...@stellaliang21 I went another route. I downloaded and extended an npm package called eslint-config-hackreactor and that is all I had in my eslintrc file. Something like this : 
module.exports = {
    extends: './node_modules/eslint-config-hackreactor/index.js'
};
 
To be sure it works I also have  "es-linst-config-hackreactor" with a link pointing to the github file of that package  in my package.json like this: 
 "dependencies": {
    "eslint": "^5.15.3",
    "eslint-config-hackreactor": "github:reactorcore/eslint-config-hackreactor"
}